commit aa5c06828826c99cdf041f50ca6f95555608d912
Author: Marco Mascheroni <marco.mascheroni@cern.ch>
Date:   Mon Aug 31 21:23:50 2015 +0200

    pycurl_manager: really decode body data
    
    The bug was introduced by this commit
    https://github.com/dmwm/WMCore/commit/8b73dd43e825fd564fe09161ddfae6c6a714d351

diff --git a/src/python/WMCore/Services/pycurl_manager.py b/src/python/WMCore/Services/pycurl_manager.py
index 385ca1f..be95e7e 100644
--- a/src/python/WMCore/Services/pycurl_manager.py
+++ b/src/python/WMCore/Services/pycurl_manager.py
@@ -138,7 +138,7 @@ class RequestHandler(object):
                         % (str(exc), type(data))
                 logging.warning(msg)
                 return data
-            return data
+            return res
         else:
             return data
 
